content/api/3-postman-collection.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Postman Collection"
+metaTitle: "Postman Collection"
+metaDescription: "Spectro Cloud API Postman Collection"
+icon: ""
+hideToC: true
+fullWidth: false
+hiddenFromNav: false
+hideToCSidebar: true
+---
+import {Intro, IntroButtons} from "shared/components"
+import InfoBox from "shared/components/InfoBox"
+
+<Intro>
+
+# Postman Collection
+
+</Intro>
+
+Spectro Cloud provides a Postman collection for your convenience to help you interact with the Spectro Cloud API.
+
+## Prerequisites
+
+To interact with the API, you will need the following:
+
+* Postman application (10.2.2 or greater)
```

Comment:
```suggestion
* [Postman application](https://www.postman.com/downloads/) v10.2.2+
```
Let's add the link to Postman. All prerequisites should contain a link to the resource.